src/riak_dt.proto

```diff
@@ -124,13 +125,15 @@ message DtFetchResp {
         COUNTER = 1;
         SET     = 2;
         MAP     = 3;
+        GSET    = 4;
```

Comment:
Hey @russelldb, correct me if I'm wrong but if I'm not mistaken that's defined in the MapEntry message: 

https://github.com/basho/riak_pb/blob/0d1f48782c1af7684bccc3eb2d5c73d82286ad67/src/riak_dt.proto#L63